rclcpp/include/rclcpp/macros.hpp

```diff
@@ -19,19 +19,19 @@
  *
  * Use in the private section of the class.
  */
-#define RCLCPP_DISABLE_COPY(Class) \
-  Class(const Class &) = delete; \
-  Class & operator=(const Class &) = delete;
+#define RCLCPP_DISABLE_COPY(...) \
+  __VA_ARGS__(const __VA_ARGS__ &) = delete; \
```

Comment:
This looks weird to me. The template accepts multiple arguments now (e.g. `Foo, Bar`) which then ends up as `const Foo, Bar &`?